test/fixtures/test-runner/NODE_TEST_WORKER_ID/1.test.js

```diff
@@ -0,0 +1,7 @@
+// This file tests the `NODE_TEST_WORKER_ID` feature.
+// The test logic is implemented in `test/parallel/test-runner-cli.js`.
```

Comment:
Can you remove this line. If things get moved around, this will almost certainly be forgotten about.